helper/staking/staking.go

```diff
@@ -0,0 +1,176 @@
fffffff16905080600084815481106107ed576107ec610cdb565b5b9060005260206000200160006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555082600360008373ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002081905550505b6000600160008573ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060006101000a81548160ff0219169083151502179055506000600360008573ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002081905550600080548061092a57610929610cac565b5b6001900381819060005260206000200160006101000a81549073ffffffffffffffffffffffffffffffffffffffff02191690559055505050565b60008135905061097381610d61565b92915050565b60006020828403121561098f5761098e610d0a565b5b600061099d84828501610964565b91505092915050565b60006109b283836109be565b60208301905092915050565b6109c781610c25565b82525050565b6109d681610c25565b82525050565b60006109e782610b61565b6109f18185610b79565b93506109fc83610b51565b8060005b83811015610a2d578151610a1488826109a6565b9750610a1f83610b6c565b925050600181019050610a00565b5085935050505092915050565b6000610a47601d83610b8a565b9150610a5282610d0f565b602082019050919050565b6000610a6a601283610b8a565b9150610a7582610d38565b602082019050919050565b610a8981610c37565b82525050565b610a9881610c73565b82525050565b6000602082019050610ab360008301846109cd565b92915050565b60006020820190508181036000830152610ad381846109dc565b905092915050565b60006020820190508181036000830152610af481610a3a565b9050919050565b60006020820190508181036000830152610b1481610a5d565b9050919050565b6000602082019050610b306000830184610a80565b92915050565b6000602082019050610b4b6000830184610a8f565b92915050565b6000819050602082019050919050565b600081519050919050565b6000602082019050919050565b600082825260208201905092915050565b600082825260208201905092915050565b6000610ba682610c73565b9150610bb183610c73565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff03821115610be657610be5610c7d565b5b828201905092915050565b6000610bfc82610c73565b9150610c0783610c73565b925082821015610c1a57610c19610c7d565b5b828203905092915050565b6000610c3082610c53565b9050919050565b60006fffffffffffffffffffffffffffffffff82169050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b600080fd5b7f4f6e6c79207374616b65722063616e2063616c6c2066756e6374696f6e000000600082015250565b7f696e646578206f7574206f662072616e67650000000000000000000000000000600082015250565b610d6a81610c73565b8114610d7557600080fd5b5056fea26469706673582212200732d127c069747c9fba40cde21178279bf118074d859232ff634cd4c0c2678964736f6c63430008070033"
```

Comment:
Sadly no, the only way to inject this sort of value into the SDK is to have it hardcoded like this, since the Polygon SDK is shipped as-is.

One other approach would be to write it to disk, and read from there - but that just introduces an additional overhead every time we want to grab the bytecode.